### ISSUE TITLE
plasma5 test job broken in nixos-unstable 

### ISSUE THREAD

**Issue by hedning** [NixOS/nixpkgs#60036]
## Issue description

`kio-extras` fails to build: https://hydra.nixos.org/job/nixpkgs/trunk/kdeApplications.kio-extras.x86_64-linux

This breaks https://hydra.nixos.org/job/nixos/trunk-combined/tested#tabs-constituents

First broken: https://hydra.nixos.org/build/92448979

Looks like an issue with the `libssh` update (25ebfe22e901017fe863a0a8ea824c790dbce125):
```
CMake Error at /nix/store/06gmnf6lb9dz2jm7kjmk65plw5ck1g1z-libssh-0.8.7-dev/lib/cmake/libssh/libssh-config.cmake:11 (message):
  File or directory
  /nix/store/06gmnf6lb9dz2jm7kjmk65plw5ck1g1z-libssh-0.8.7-dev/lib/libssh.so
  referenced by variable LIBSSH_LIBRARIES does not exist !
```

cc @ttuegel @bkchr and libssh maintainer @svanderburg

**Comment by hedning** [comment]
Ah, apparently fixed by #60035

**Comment by worldofpeace** [comment]
9bfb6d535bc672f9be742c5cf1991d453a99b22e would probably fix another problem down the line.

`libssh`'s cmake file is definitely not right (https://github.com/NixOS/nixpkgs/pull/58307#issuecomment-485218238)
